Library/Formula/corectl.rb

```diff
@@ -0,0 +1,45 @@
+require "language/go"
+
+class Corectl < Formula
+  desc "CoreOS over OS X made very simple"
+  homepage "https://github.com/TheNewNormal/corectl"
```

Comment:
@mikemcquaid let's say that the [original](https://github.com/coreos/coreos-xhyve) upstream got way _too_ quiet and had other priorities [i kept this as a _fork_to preserve commit history and also due to plain politeness, as original concept was theirs (@philips)].